.github/workflows/orca.yml

```diff
@@ -40,23 +40,33 @@ jobs:
           - ""
           # We do not run deprecated code scans since they'd scan the entire
           # codebase (since the SUT is the project template).
-        php-version: [ "7.4" ]
+        php-version: [ "8.1" ]
         include:
+          - orca-job: INTEGRATED_TEST_ON_OLDEST_SUPPORTED
+            php-version: "7.4"
```

Comment:
@mikemadison13 I think if we are testing drupal-recommended-project for Drupal core 9.x with PHP 7.4 & 8.0, then shouldn't we need to change PHP require in composer.json like:
```
"require": {
  "php": "^7.4 || ^8.0",
}
```